mungegithub/mungers/approval-handler.go

```diff
@@ -70,7 +71,10 @@ func (h *ApprovalHandler) Initialize(config *github.Config, features *features.F
 func (*ApprovalHandler) EachLoop() error { return nil }
 
 // RegisterOptions registers options for this munger; returns any that require a restart when changed.
-func (*ApprovalHandler) RegisterOptions(opts *options.Options) sets.String { return nil }
+func (a *ApprovalHandler) RegisterOptions(opts *options.Options) sets.String {
+	opts.RegisterBool(&a.issueRequired, "issue-required", false, "flag indicating if all PRs must be associated with an issue")
```

Comment:
might or not be related but, the mungebot is accumulating more and more command-line arguments, which don't have a munger-specific semantic. Should we make this more explicit, i.e. `approval-requires-issue` ?

EDIT: Re-reading, even the description of the flag is not super clear it's about approval.